docs/integrations/sources/smartsheets.md

```diff
@@ -86,7 +86,8 @@ To setup your new Smartsheets source, Airbyte will need:
 
```

Comment:
add info that connector support incremental syncs now

Reply:
done in the next PR https://github.com/airbytehq/airbyte/pull/12203/files